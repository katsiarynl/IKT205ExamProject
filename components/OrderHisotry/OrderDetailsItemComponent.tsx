import { View, Text } from "react-native";
import React from "react";

export default function OrderDetailsItemComponent(props) {
  return (
    <View>
      <Text>{props.name}</Text>
      <Text> {props.description}</Text>
    </View>
  );
}
