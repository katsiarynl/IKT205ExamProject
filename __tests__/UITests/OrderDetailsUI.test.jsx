import OrderDetailsItemComponent from "../../components/OrderHisotry/OrderDetailsItemComponent";
import { create } from "react-test-renderer";

const tree = create(
  <OrderDetailsItemComponent
    name="Restaurant Name"
    description="Restaurant Description"
  />
);

test("Snapshot for OrderDetailsItemComponent", () => {
  expect(tree).toMatchSnapshot();
});
