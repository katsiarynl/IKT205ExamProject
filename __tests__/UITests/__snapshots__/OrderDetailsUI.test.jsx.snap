// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View>
  <Text>
    Restaurant Name
  </Text>
  <Text>
     
    Restaurant Description
  </Text>
</View>
`;
